.github/workflows/jobs-integration-test.yml

```diff
@@ -0,0 +1,57 @@
+name: Integration test
+on:
+  workflow_call:
+    inputs:
+      environment_name:
+        type: string
+        required: true
+        description: The name of the environment.
+      project_prefix_name:
+        type: string
+        required: true
+        description: The prefix name of the project
+    secrets:
+      AZURE_CREDENTIALS:
+        required: true    
+jobs:
+
+  IntegrationTest:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: Login to Azure
+      uses: Azure/login@v1
+      with:
+        creds: ${{ secrets.AZURE_CREDENTIALS }}
+
+    - name: Get secrets from key vault
+      env:
+        keyvault_name: ${{ inputs.project_prefix_name }}-${{ inputs.environment_name }}-kv
+      uses: Azure/get-keyvault-secrets@v1
+      with: 
+        keyvault: ${{ env.keyvault_name }}
+        secrets: 'CentOpsAdminApiKey, MockBot1ApiKey'
+      id: appSecrets
+    
+    - name: Checkout Test repo
+      uses: actions/checkout@v3
+      with:
+        repository: buerokratt/Tests
+        path: Tests
+        
+    - name: Build integration test
+      run: dotnet build Tests/src/Tests.IntegrationTests/*.csproj
+       
+    - name: Execute integration test
+      env: 
+        CentOpsApiKey: ${{ steps.appSecrets.outputs.CentOpsAdminApiKey }}
+        Bot1ApiKey: ${{ steps.appSecrets.outputs.MockBot1ApiKey }}
+        Bot1Url: "https://${{ inputs.project_prefix_name }}-${{ inputs.environment_name }}-aks-ingress.westeurope.cloudapp.azure.com/mock-bot"
+        CentOpsUrl: "https://${{ inputs.project_prefix_name }}-${{ inputs.environment_name }}-aks-ingress.westeurope.cloudapp.azure.com/centops"
+        ClassifierInternalUrl: "https://mock-classifier-service.applications.svc.cluster.local"
```

Comment:
When running in 'Fixture Free' mode - are these internal Urls even needed?

Reply:
Sorry, I missed this before it merged (I think the policy is not fully setup on this repo). It's a good question, I'm not sure of the answer but will take a look and create backlog items if there is anything to change.